.github/workflows/publish-docker-images.yml

```diff
@@ -79,15 +78,47 @@ jobs:
           subject-digest: ${{ steps.push_cli.outputs.digest }}
           push-to-registry: true
 
+      - name: Set Up Container Image Tags for Boring CLI Container
+        env:
+          IMAGE_REPO: ${{ env.REGISTRY }}/${{ env.ZITI_CLI_IMAGE }}
+          IMAGE_TAG: ${{ env.ZITI_CLI_TAG }}-boring
```

Comment:
why all the `-boring` instead of `-fips`? i thought you were using `-fips`?

Reply:
It seems more accurate and descriptive of how this artifact differs. Either would work just fine, and now's the time to decide on a consistent representation. I vote for `openziti/ziti-controller:1.6.2-boring`, but I'll make the change to `openziti/ziti-controller:1.6.2-fips` if you feel strongly about it.

Reply:
it means people will have to change the image they pull if WE change from boring. which is why i don't like -boring.